### PR TITLE
Clarify setup docs

### PR DESCRIPTION
**Pull request by sjspielman**
As we discussed earlier, this PR makes an attempt to clarify the technical setup docs to better differentiate between local and LSfR setup. I consolidated both the `technical-setup/index.md` and `technical-setup/environment-setup/index.md` files into one and deleted the latter, with some reorganization as part of it. Let me know what you think of these changes!

### CODE REVIEW

File: docs/software-platforms/aws/index.md [hunk 1]
```diff
@@ -31,5 +31,5 @@ Please refer to these pages to learn more about setting up and using LSfR:
 - [Accessing a virtual computer](./accessing-vcs.md)
 - [Working with LSfR volumes](./working-with-volumes.md)
 - [Developing analyses on LSfR](./starting-development-on-lsfr.md)
-  - You can also find instructions for setting up virtual computers for development on this page.
+    - You can also find instructions for setting up your virtual computers for OpenScPCA development on this page.
```
Comment (jashapiro) [inline docs/software-platforms/aws/index.md:34]: I don't think we need both changes... I also wonder if we want to clarify what we mean by "virtual computer" here. I have a different meaning in my head, so I think we should clarify that these are "in the cloud" somewhere in this section/title. 


```suggestion
    - You can also find instructions for setting up virtual computers for OpenScPCA development on this page.
```

File: docs/technical-setup/environment-setup/index.md [hunk 2]
```diff
@@ -4,6 +4,7 @@ This section of the documentation walks you through additional setup steps that
 
 This section assumes that you have already taken these setup steps:
 
+- If you are working on a Windows computer, [install WSL 2](../install-wsl-2.md)
 - Download, install, and set up your [Git client](../install-a-git-client.md)
 - [Fork the `AlexsLemonade/OpenScPCA-analysis` repository](../fork-the-repo.md)
 - [Clone your fork](../clone-the-repo.md) to your computer
```
Comment (jashapiro) [inline docs/technical-setup/environment-setup/index.md:10]: Should these all be in past tense?

File: docs/technical-setup/index.md [hunk 3]
```diff
@@ -1,17 +1,36 @@
...
+All of these steps are required setup when working locally on your computer (or server).
+
+If you are planning to work on a Lightsail for Research instance, please instead refer to the [Lightsail for Research documentation](../software-platforms/aws/index.md#lightsail-for-research-virtual-computing-with-aws) for setup instructions.
+
+
+## Local setup steps
+
+To prepare to contribute to OpenScPCA, please do the following in order:
+
+1. If you are on a Windows machine, you will need to [install WSL 2](./install-wsl-2.md), as described below.
```
Comment (jashapiro) [inline docs/technical-setup/index.md:14]: They are all needed, so for paralellism, removing a clause.
```suggestion
1. If you are on a Windows machine, [install WSL 2](./install-wsl-2.md), as described below.
```

**Review by jashapiro — COMMENTED** [review]
The organization here looks fine overall, but I had a few suggestions about some modifications for clarity that I think are worth doing while we are making edits.

File: docs/technical-setup/environment-setup/index.md (hunk 2 above)
Comment (sjspielman) [reply docs/technical-setup/environment-setup/index.md:10]: Ugh I meant to delete this file..!

**Comment by sjspielman** [comment]
Alright, that makes sense that I broke a bunch of links with https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/415/commits/4e10e4fac798ec0f4396bf08faf99958f8b2ff1c. @dvenprasad, let me know what you think about removing this file first, and then I'll fix the links or revert!

**Review by dvenprasad — APPROVED** [review]
The distinction between local and LsFR is clearer now! 
🦇 🦇 🦇

**Pull request merged**